source/utilities.tex

```diff
@@ -15121,8 +15122,8 @@
 
 \pnum
 \begin{note}
-The types deduced by the deduction guides for \tcode{function}
-might change in future revisions of \Cpp{}.
+It is possible that the types deduced by the deduction guides for \tcode{function}
+change in future revisions of \Cpp{}.
```

Comment:
I don't think this has the same implications. @jwakely, what about: "The types deduced by the deduction guides for `function` have been identified as candidates for being changed in a future revision of \Cpp{}."? Is that accurate?

Reply:
Yes, I think so.

Reply:
fixed